doc/guides/conflicts.md

```diff
@@ -0,0 +1,125 @@
+## Conflict resolution
+
+## Prerequisites:
+
+- GraphQL server with resolvers
+- Database or any other form of data storage that can cause data conflicts
+
+## Pluggable conflict resolution
```

Comment:
Minor thing but headings should always be captialized with the exception of words like `the, and, of, for, in, at` etc. (unless they are at the beginning of the heading of course).

```suggestion
## Pluggable Conflict Resolution
```